dbmeta/codegen.go

```diff
@@ -501,6 +501,7 @@ func (c *Config) CreateContextForTableFile(tableInfo *ModelInfo) map[string]inte
 	var modelInfo = map[string]interface{}{
 		"StructName":         tableInfo.StructName,
 		"TableName":          tableInfo.TableSchemaAndName.TableName,
+		"TableSchema":        tableInfo.TableSchemaAndName.TableSchema,
```

Comment:
hmm do you need to add this? Why not jus use TableSchemaAndName.Schema?

Reply:
Just did in analogy with the TableName parameter above. But your suggestion would also work

Reply:
`TableName` was in the original repo. `TableSchema` was added by me. I didn't remove `TableName` for backward compatibility.